src/hooks/useSafeNotifications.ts

```diff
@@ -31,33 +34,33 @@ const useSafeNotifications = (): void => {
       return
     }
 
-    const isOldSafe = isOldestVersion(version)
+    const isOldSafe = !isValidSafeVersion(version)
 
     const id = dispatch(
       showNotification({
         variant: 'warning',
         groupKey: 'safe-outdated-version',
 
         message: isOldSafe
-          ? `Safe version ${version} is not supported by this web app anymore. We recommend using the command line interface instead.`
+          ? `Safe version ${version} is not supported by this web app anymore. You can update your Safe via the old web app here.`
           : `Your Safe version ${version} is out of date. Please update it.`,
 
-        link: isOldSafe
-          ? CLI_LINK
-          : {
-              href: {
+        link: {
+          href: isOldSafe
+            ? `https://gnosis-safe.io/app/${chain?.shortName}:${safeAddress}/settings/details?no-redirect=true`
```

Comment:
Please make the old URL a constant. The safe query param can be taken from the router instead of concatenating chainId and safeAddress.